api/nodes/serializers.py

```diff
@@ -72,8 +45,26 @@ class NodeSerializer(JSONAPISerializer):
                                                             'public and private nodes. Administrators on a parent '
                                                             'node have implicit read permissions for all child nodes',
                               )
-    # TODO: finish me
 
+    children = HyperlinkedIdentityFieldWithMeta(view_name='nodes:node-children', lookup_field='pk', link_type='related',
+                                                lookup_url_kwarg='node_id', meta={'count': 'get_node_count'})
+
+    contributors = HyperlinkedIdentityFieldWithMeta(view_name='nodes:node-contributors', lookup_field='pk', link_type='related',
+                                                    lookup_url_kwarg='node_id', meta={'count': 'get_contrib_count'})
+
+    files = HyperlinkedIdentityFieldWithMeta(view_name='nodes:node-files', lookup_field='pk', lookup_url_kwarg='node_id',
+                                             link_type='related')
+
+    node_links = HyperlinkedIdentityFieldWithMeta(view_name='nodes:node-pointers', lookup_field='pk', link_type='related',
+                                                  lookup_url_kwarg='node_id', meta={'count': 'get_pointers_count'})
+
+    parent = HyperlinkedIdentityFieldWithMeta(view_name='nodes:node-detail', lookup_field='parent_id', link_type='self',
+                                              lookup_url_kwarg='node_id')
+
+    registrations = HyperlinkedIdentityFieldWithMeta(view_name='nodes:node-registrations', lookup_field='pk', link_type='related',
+                                                     lookup_url_kwarg='node_id', meta={'count': 'get_registration_count'})
+
+    # TODO: finish me
```

Comment:
I had written this TODO while prototyping; at the time, I had only implemented a small subset of node fields. It can be removed now.